server/lib/data_import/my_anime_list.rb

```diff
@@ -0,0 +1,43 @@
+module DataImport
+  class MyAnimeList
+    MDL_HOST = "https://hbv3-mal-api.herokuapp.com/2.1" # 2.1 is the api version, MAYBE add graceful decay to check from v1 if it can't find?
```

Comment:
I don't think we need a decay.  We should be very clear which version we need, and since we have our own instance this isn't a problem.  If anything, this number should only ever increase.